Makefile

```diff
@@ -298,11 +302,23 @@ fmt-check: fmt
 	@$(call print, "Checking fmt results.")
 	if test -n "$$(git status --porcelain)"; then echo "code not formatted correctly, please run `make fmt` again!"; git status; git diff; exit 1; fi
 
-#? lint: Run static code analysis
-lint: docker-tools
+check-go-version-yaml:
+	@$(call print, "Checking for target Go version (v$(GO_VERSION)) in YAML files (*.yaml, *.yml)")
+	./tools/check-go-version-yaml.sh $(GO_VERSION)
+
+check-go-version-dockerfile:
+	@$(call print, "Checking for target Go version (v$(GO_VERSION)) in Dockerfile files (*Dockerfile)")
+	./tools/check-go-version-dockerfile.sh $(GO_VERSION)
```

Comment:
I noticed this only after the PR in `taproot-assets` was merged: Shouldn't this file also go into the `scripts` folder? Same with the one above. The `tools` folder is specifically for binaries that we install using a separate `go.mod` file.

Reply:
I'll move in tap also.